tests/test_monitor.py

```diff
@@ -34,3 +35,47 @@ def test_monitor():
     assert set(last_logline.keys()) == {'l', 't', 'r'}, "Incorrect keys in monitor logline"
     file_handler.close()
     os.remove(mon_file)
+
+def test_monitor_load_results(tmp_path):
+    """
+    test load_results on log files produced by the monitor wrapper
+    """
+    env1 = gym.make("CartPole-v1")
+    env1.seed(0)
+    mon_file1 = os.fspath(tmp_path / "stable_baselines-test-{}.monitor.csv".format(uuid.uuid4()))
```

Comment:
`os.fspath` is not available in python 3.5... (is this why you did not use `os.path.join` ?)

Reply:
Ah, my bad. The underlying problem is that load_results assumes the path is a string object by using `endswith` on it. I will find another solution.